tests/ctst/common/common.ts

```diff
@@ -342,8 +339,13 @@ Then('kafka consumed messages should not take too much place on disk', { timeout
                 .filter(t => (t.includes(this.parameters.InstanceID) &&
                 !ignoredTopics.some(e => t.includes(e))));
 
+            timeoutID = setTimeout(() => {
+                assert.fail('Kafka cleaner did not clean the topics within the expected time');
+            }, (topics.length || 1) * checkInterval * 5); // Timeout after 5 Kafka cleaner intervals
```

Comment:
why do we increase the timeout by the number of topics?
- the timeout is based on kafka-cleaner period, and we check every (remaining) topic on each iteration: so number of topics should not increase the timeout...
- move the timeout here prevents catching issues with kafka connection (`listTopics()` call above) which could get stuck...

Reply:
I adapted the logic, it's true we do not need to wait for each topic, but for sure we needed to wait more, as the cleaning might be slower in the CI due to other tests jeopardizing resources temporarily...